txmatching/database/db_migrations/0033.make-txm-event-id-required-in-parsing-issue.sql

```
@@ -0,0 +1,3 @@
+
+
+ALTER TABLE parsing_issue ALTER column txm_event_id SET NOT NULL;
```

newline and comments at the beginning of the file are missing - please check other sql files.. 

@kubantjan i dont suppose there are any parsing_issues without a txm_event_id value, but just to be precautious, shouldnt we set a default value here to overcome any problems? (more precisely i am asking - which value)

---

maybe there can be smth like `clean` django function that says "have at least one of these attributes set"?

---

So there is couple of things:
1) Check please the stg and demo servers but I think there should be no parsing issues without txm event. I will check production after that. I would not put any default and rather let the migration fail
2) yes not all should be required .and this is the case now, donor_id and recipient id are not required. Quick search found smth like that: https://stackoverflow.com/questions/5447880/adding-an-one-out-of-two-not-null-constraint-in-postgresql we can add this condition

---

staging and demo checked, should have no parsing issues with txm_event_id null @kubantjan